### ISSUE TITLE
numpy error

### ISSUE THREAD

**Issue by KikeVen** [davidberenstein1957/classy-classification#50]
I installed it, and when I ran it I got the following error:

```bash
ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
```

Thank you

**Comment by davidberenstein1957** [comment]
Hi @KikeVen , I will have a look this wekend.

**Comment by davidberenstein1957** [comment]
@KikeVen could you add some more context about your environment and such?

**Comment by davidberenstein1957** [comment]
This should be solved in version 1.0.1

**Comment by KikeVen** [comment]
Thank you for your quick response.

- I am running a Windows 11 7i with 32 RAM
- venv 3.10

When I `pip install classy-classification` it installs version `1.0.0`. I installed it yesterday. Should I do an update?

I kind of found a workaround:

```bash
pip install classy-classification
pip install --upgrade numpy
pip install --upgrade pandas scikit-learn
```

> I still get a warning when installing the dependencies, but at least it runs now.

Let me know if you think I should update.

Thank you again
